### PR TITLE
Uploader images recepisse

### PR DESCRIPTION
**Pull request by eliegaboriau**
#### :tophat: Description
*Please describe your pull request.*

This PR changes the way images are handled in the recepisse.
They now mus be uploaded in the organization's appearance settings, and are displayed in the recepisse.

#### Testing
*Describe the best way to test or validate your PR.*

Example:
- Go to the organization appearance settings
- Add cachet image and signature image
- Publish a petition you created
- Click on "voir le recepisse"
- See your images

#### Tasks
- [x] Add specs

### CODE REVIEW

**Review by luciegrau — APPROVED** [review]
LGTM thanks Elie ! 🍂

File: db/migrate/20230404130043_add_secrets_image_to_organization.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddSecretsImageToOrganization < ActiveRecord::Migration[6.1]
```
Comment (Quentinchampenois) [inline db/migrate/20230404130043_add_secrets_image_to_organization.rb:3]: We should allow reversibility of this migration by adding columns in `up` method and removing columns in `down` method

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Nice work it works great, I think it is the best alternative we have ! 

I just have a little change request about the migration, we should ensure now that we can revert migration before running in database

**Review by Quentinchampenois — APPROVED** [review]
Nice feature !

**Pull request merged**